scripts/wasm-all.sh

```diff
@@ -4,60 +4,48 @@
 if [[ $(arch) == "arm64" ]]; then
 image="cosmwasm/rust-optimizer-arm64"
 workspace_image="cosmwasm/workspace-optimizer-arm64"
-abstract_image="abstractsdk/workspace-optimizer-arm64"
+abstract_image="abstractmoney/workspace-optimizer-arm64"
```

Comment:
I guess workspace image not used anymore?

Reply:
It's still the workspace image, it's just a fork that allows featured builds!

Reply:
I meant if we plan to use `abstract_image` over `workspace_image` why keep both :laughing: